src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java

```diff
@@ -0,0 +1,16 @@
+package opgg.weba.JamPick.common;
+
+import opgg.weba.JamPick.common.GetLocale;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class GetLocaleTest {
+
+    @Test
+    public void GetAcceptLanguage() {
+
+        Assertions.assertThat(GetLocale.getLocale().toString()).isEqualTo("ko_KR");
```

Comment:
You can use also
`Assertions.assertThat(GetLocale.getLocale()).isEqualTo(Locale.KOREA);`

Reply:
As i know, `GetLocale` return accept-language of request header. But test hasn't test code about http request or request header. Is it okay?

Reply:
I agree with that. I'll modify logic. thanks!

Reply:
There were some problems when test with Mockmvc.
So i made TestController that return only locale value and test with postman.

Result is below

![image](https://user-images.githubusercontent.com/62228195/129156898-7ef70392-8723-4bd9-9ff5-c7abcd52a846.png)

But, i won't add TestController.

Reply:
It is okay that edit this test in future. If you want to do other task at first, merge this PR.

I approved PR, thanks